.travis.yml

```diff
@@ -16,6 +16,7 @@ language: go
 branches:
   only:
     - master
+    - release-v2.1
```

Comment:
this might fail to rebase? it was included in #944 too

Reply:
Yes we don't need this anymore